chapters.md

```diff
@@ -17,7 +17,62 @@ As an example, consider a `Parent ChapterAtom Element` that has its `ChapterFlag
 
 Three `Edition Flags` are defined to describe the bevahior of the `EditionEntry Element`: `EditionFlagHidden`, `EditionFlagDefault` and `EditionFlagOrdered`.
```

Comment:
>  If a `ChapterAtom Element` is a `Child Element` of another `ChapterAtom Element` which has a `Chapter Flag` set to `true`

Maybe better if...

If a `ChapterAtom Element` is **the** `Child Element` of another `ChapterAtom Element` **with** a `Chapter Flag` set to `true`, then the `Child ChapterAtom Element` MUST be **interpreted** as having its `Chapter Flag` set to `true`. If a `ChapterAtom Element` is **the** `Child Element` of another `ChapterAtom Element` **with** a `Chapter Flag` set to `false` or **if** the `ChapterAtom Element` does not have a `ChapterAtom Element` as its `Parent Element`, then it MUST be **interpreted** according to its own `Chapter Flag`.